arlo-client/src/components/Audit/RoundManagement/QRs.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import QRCode from 'qrcode.react'
+import styled from 'styled-components'
+
+const QRroot = styled.div`
+  display: none;
+`
+
+const QRs: React.FC<{ electionId: string; boardIds: string[] }> = ({
+  electionId,
+  boardIds,
+}: {
+  electionId: string
+  boardIds: string[]
+}) => {
+  return (
+    <QRroot id="qr-root">
+      {boardIds.map(id => (
+        <span key={id} id={`qr-${id}`}>
+          <QRCode
+            value={`${window.location.origin}/election/${electionId}/audit-board/${id}`}
```

Comment:
this should link to `/auditboard/${auditBoard.passphrase}` because hitting that url will initiate the audit board's session (which is needed to authorize requests to the audit board API)